package.json

```diff
@@ -4,8 +4,9 @@
 	"description": "Increase or decrease heading depth",
 	"main": "index.js",
 	"engines": {
-		"node": ">=8"
+	  "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

Comment:
Use tab

Reply:
Whoops! I thought my IDE would've handled that for me. Will update